introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello World! My name is Michelle, and I'm learning GitHub.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?